osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerMatchSubScreen.cs

```diff
@@ -223,12 +226,17 @@ protected override void LoadComplete()
         /// Opens the song selection screen to add or edit an item.
         /// </summary>
         /// <param name="itemToEdit">An optional playlist item to edit. If null, a new item will be added instead.</param>
-        internal void OpenSongSelection(long? itemToEdit = null)
+        internal void OpenSongSelection(PlaylistItem itemToEdit = null)
         {
             if (!this.IsCurrentScreen())
                 return;
 
-            this.Push(new MultiplayerMatchSongSelect(Room, itemToEdit));
+            int id = itemToEdit?.Beatmap.OnlineID ?? Room.Playlist.LastOrDefault()!.Beatmap.OnlineID;
```

Comment:
I think you meant `Last()` there, if it's guaranteed to not be null.
```suggestion
            int id = itemToEdit?.Beatmap.OnlineID ?? Room.Playlist.Last().Beatmap.OnlineID;
```